indico/modules/categories/operations.py

```diff
@@ -51,22 +51,16 @@ def move_category(category, target_category):
 
 
 @make_interceptable
-def update_category(category, data, skip=()):
-    assert set(data) <= {
-        'title', 'description', 'timezone', 'suggestions_disabled', 'is_flat_view_enabled',
-        'event_message_mode', 'event_message', 'notify_managers', 'event_creation_notification_emails',
-        *skip
-    }
+def update_category(category, data, _extra_log_fields, skip=()):
```

Comment:
This is now a required argument, that's not OK - we do not want to always pass this when calling the method in the core.

```suggestion
def update_category(category, data, *, skip=(), _extra_log_fields=None):
```